### PR TITLE
fix(DB/Creatures): Corrected Snivel Rustrocket's position.

### LINKED ISSUE #10658: [Love is in the Air] Snivel Rustrocket is two steps away from where it should be
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour

Original report: https://github.com/chromiecraft/chromiecraft/issues/2991

The NPC is between the boxes.

imagen

### Expected Blizzlike Behaviour

He should be ahead of the boxes.

WorstBugReportEver

### Source

[Wowhead](https://www.wowhead.com/quest=24657/a-friendly-chat#screenshots), dates in the screenshot above.

### Steps to reproduce the problem

DB GUID: 244630, current GUID: 1410004.
Current Entry: 37715 of (37715, 0, 0).
DisplayID: 30725 (Native: 30725).

* `go creature 244630`

### Extra Notes

This is a really minor thing but I noticed it.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/6f20b9ece71984f1a11469e65ede8accda05653e

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-save-inventory](https://github.com/Nyeriah/mod-save-inventory)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #10658

## Issues Addressed:

- Closes #10658
- Closes https://github.com/chromiecraft/chromiecraft/issues/2991

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

`.event start 8`
`.go creature 244630`

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
What is the source of the new position?

**Comment by Kofmy** [comment]
degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates degenerates

File: data/sql/updates/pending_db_world/rev_1678624730000970200.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `creature` SET `position_x`=-8399.23, `position_y`=1246.33, `position_z`=5.31357 WHERE `guid`=244630;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1678624730000970200.sql:2]:
```suggestion
UPDATE `creature` SET `position_x`=-8399.23, `position_y`=1246.33, `position_z`=5.31357 WHERE `guid`=244630 AND `id1`=37715;
```

**Comment by Gultask** [comment]
> What is the source of the new position?

It's Mangos

**Review by Gultask — APPROVED** [review]

**Pull request merged**